frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -137,42 +129,67 @@ export function SmartScalar({
           />
         </span>
       </ScalarContainer>
-      {isPeriodVisible(innerHeight) && (
-        <ScalarPeriod lines={1} period={display.date} />
-      )}
+      {isPeriodVisible(innerHeight) && <ScalarPeriod period={display.date} />}
       {comparisons.map((comparison, index) => (
-        <PreviousValueWrapper key={index} data-testid="scalar-previous-value">
+        <Box maw="100%" key={index} data-testid="scalar-previous-value">
           <PreviousValueComparison
             comparison={comparison}
             fontFamily={fontFamily}
             formatOptions={formatOptions}
             width={width}
           />
-        </PreviousValueWrapper>
+        </Box>
       ))}
     </ScalarWrapper>
   );
 }
 
-function ScalarPeriod({ lines = 2, period, onClick }) {
+function ScalarPeriod({ period, onClick }) {
   return (
-    <ScalarTitleContainer data-testid="scalar-period" lines={lines}>
-      <ScalarPeriodContent
+    <ScalarTitleContainer data-testid="scalar-period" lines={1}>
+      <Text
+        component="h3"
```

Comment:
I see you changed from `order={3}` to `component=h3 + fw=700`. Why is that? Are they different?

Reply:
@WiNloSt Yes, `Title` has a couple of Mantine styling that we don't want, but using `component={h3}` only brings in the tag without applying the styles.

Reply:
Gotcha. Thanks!